addons/bestja_frontend_fpbz/templates/pages/home_page.xml

```diff
@@ -0,0 +1,357 @@
+                                            <a href="/o/ruda-slaska" class="ruda-slaska">
+                                                <span class="nav-map-point nav-map-point-dot"></span>
+                                                <span class="nav-map-point-label">Chorzów</span>
+                                            </a>
+                                        </li>
+                                        <li style="top: 340px; left: 255px;" id="krakow">
+                                            <a href="/o/krakow" class="krakow">
+                                                <span class="nav-map-point nav-map-point-dot"></span>
+                                                <span class="nav-map-point-label">Kraków</span>
+                                            </a>
+                                        </li>
+                                        <li style="top: 318px; left: 345px;" id="rzeszow">
+                                            <a href="/o/rzeszow" class="rzeszow">
+                                                <span class="nav-map-point nav-map-point-dot"></span>
+                                                <span class="nav-map-point-label">Rzeszów</span>
+                                            </a>
+                                        </li>
+                                    </ul>
+                                    <div class="col-md-6 mt16 text-right-only-desktop">
+                                        <p>Twoja organizacja chciałaby z nami współpracować?</p>
+                                    </div>
+                                    <div class="col-md-6 mt16 text-left-only-desktop">
+                                        <p>
+                                            <a class="color_orange" href="">Dowiedz się więcej</a> lub 
+                                            <a class="color_orange" href="">zostań organizacją partnerską</a>
+                                        </p>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </section>
+                    <section class="oe_snippet_body">
+                        <div class="container-fluid quote_div open_sans_font">
+                            <div class="row">
+                                <blockquote>
+                                    <p>
+                                        <span class="big_apostrophe">" </span>Wolontariat daje mi przede wszystkim satysfakcję z pomagania, 
+                                        a także możliwość kreatywnego działania. Poznaję nowych ludzi, 
+                                        uczę się pracy w zespole i rozwijam swoje zainteresowania.
+                                        <span style="font-family: 'Arial Bold', Arial; font-weight: 700; font-style: normal; font-size: 48px;" class="" id="cache59">" </span>
```

Comment:
Instead of that strange `<span>` tag, we should probably use the `big_apostrophe` class.